### DISCUSSION TITLE
How to complete Setup Streampipes with docker Compose or in kubernetes

### DISCUSSION THREAD

**Discussion by MichaelKoch11** [apache/streampipes#2989] [Q&A]
I dont find an actual Example how to start streampipes with the flink wrapper. Maybe someone can link our add an complete docker-compose or values.yaml for test start.

**Comment by tenthe** [comment]
Hi @MichaelKoch11,

thanks for initiating this discussion!

I haven't used the Flink components for a while. Additionally, there's an open issue (#1740) about updating the Flink version to support Java 17.

@obermeier, do you have any updates or further information on this topic?

Also, the extensions were moved to an experimental module. Is anyone currently using this module? How should we proceed with Flink support, and is anyone interested in working on it?
